com/github/mashlol/MySQL.java

```diff
@@ -133,8 +168,11 @@ public void close() {
 		try {
 			con.close();
 		} catch (SQLException e) {
-			System.out.println("A bad call to mysql.close() occurred");	
-			e.printStackTrace();
+                            System.out.println("[StockMarket] " + "General Database error. Could not close SQL connection after use! Enable debug-mode for more info.");
+                            if (StockMarket.debugMode == true) {
+                                e.printStackTrace();
+                            }	
+                            dbstatus = false;
```

Comment:
You mean like this - sort of? (Syntax might not be 100%): 

	public void throwMsg($type, $msg, $fatal) {
                    log.$type("[StockMarket] " + msg);
                    if ($type == 'severe') {
	                    this.disablePlugin();
                    }
                    return;
	}